samples/sampler/stories/default/FanSpeedControl.js

```diff
@@ -1,6 +1,6 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import {action} from '@enact/storybook-utils/addons/actions';
-import {number, select} from '@enact/storybook-utils/addons/knobs';
+import {boolean, number, select} from '@enact/storybook-utils/addons/knobs';
```

Comment:
Why did you change sampler in this PR?
Is this needed by your change?
Let's do this when we add `disabled` prop for these components.

Reply:
Reverted.